core/src/key/category.rs

```diff
@@ -104,69 +115,71 @@ pub enum KeyCategory {
 	///
+	/// ------------------------------
+	///
 	/// crate::key::thing                    /*{ns}*{db}*{tb}*{id}
 	Thing,
 	///
+	/// ------------------------------
+	///
 	/// crate::key::graph                    /*{ns}*{db}*{tb}~{id}{eg}{fk}
 	Graph,
 }
 
-impl Display for KeyCategory {
+impl Display for Category {
 	fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
 		let name = match self {
-			KeyCategory::Unknown => "Unknown",
-			KeyCategory::Root => "Root",
-			KeyCategory::Access => "Access",
-			KeyCategory::Heartbeat => "Heartbeat",
-			KeyCategory::Node => "Node",
-			KeyCategory::NamespaceIdentifier => "NamespaceIdentifier",
-			KeyCategory::Namespace => "Namespace",
-			KeyCategory::User => "User",
-			KeyCategory::NodeRoot => "NodeRoot",
-			KeyCategory::NodeLiveQuery => "NodeLiveQuery",
-			KeyCategory::NamespaceRoot => "NamespaceRoot",
-			KeyCategory::DatabaseAlias => "DatabaseAlias",
-			KeyCategory::DatabaseIdentifier => "DatabaseIdentifier",
-			KeyCategory::DatabaseLogAlias => "DatabaseLogAlias",
-			KeyCategory::NamespaceAccess => "NamespaceAccess",
-			KeyCategory::NamespaceUser => "NamespaceUser",
-			KeyCategory::DatabaseRoot => "DatabaseRoot",
-			KeyCategory::DatabaseAccess => "DatabaseAccess",
-			KeyCategory::DatabaseAnalyzer => "DatabaseAnalyzer",
-			KeyCategory::DatabaseFunction => "DatabaseFunction",
-			KeyCategory::DatabaseLog => "DatabaseLog",
-			KeyCategory::DatabaseModel => "DatabaseModel",
-			KeyCategory::DatabaseParameter => "DatabaseParameter",
-			KeyCategory::DatabaseTable => "DatabaseTable",
-			KeyCategory::DatabaseTableIdentifier => "DatabaseTableIdentifier",
-			KeyCategory::DatabaseTimestamp => "DatabaseTimestamp",
-			KeyCategory::DatabaseUser => "DatabaseUser",
-			KeyCategory::DatabaseVersionstamp => "DatabaseVersionstamp",
-			KeyCategory::TableRoot => "TableRoot",
-			KeyCategory::TableEvent => "TableEvent",
-			KeyCategory::TableField => "TableField",
-			KeyCategory::TableView => "TableView",
-			KeyCategory::IndexDefinition => "IndexDefinition",
-			KeyCategory::TableLiveQuery => "TableLiveQuery",
-			KeyCategory::IndexRoot => "IndexRoot",
-			KeyCategory::IndexTermDocList => "IndexTermDocList",
-			KeyCategory::IndexBTreeNode => "IndexBTreeNode",
-			KeyCategory::IndexTermDocFrequency => "IndexTermDocFrequency",
-			KeyCategory::IndexDocKeys => "IndexDocKeys",
-			KeyCategory::IndexTermList => "IndexTermList",
-			KeyCategory::IndexBTreeNodeDocLengths => "IndexBTreeNodeDocLengths",
-			KeyCategory::IndexOffset => "IndexOffset",
-			KeyCategory::IndexBTreeNodePostings => "IndexBTreeNodePostings",
-			KeyCategory::IndexFullTextState => "IndexFullTextState",
-			KeyCategory::IndexBTreeNodeTerms => "IndexBTreeNodeTerms",
-			KeyCategory::IndexTerms => "IndexTerms",
-			KeyCategory::Index => "Index",
-			KeyCategory::ChangeFeed => "ChangeFeed",
-			KeyCategory::Thing => "Thing",
-			KeyCategory::Graph => "Graph",
+			Category::Root => "Root",
+			Category::Access => "Access",
```

Comment:
We could even use Self::...

Reply:
Fixed in https://github.com/surrealdb/surrealdb/pull/4257/commits/a26909e1346e95688f2d26ffaa2f20af4f244c8e